### ISSUE TITLE
False positive when empty tuple is used in `except` statement

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#5684]
The Python runtime allows an empty tuple to be used with an `except` statement, but pyright incorrectly flags this as an error.

```python
try:
    pass
except ():
    pass
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.322, which I just published.
